compose/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ android {
     compileOptions {
         sourceCompatibility = JavaVersion.VERSION_11
         targetCompatibility = JavaVersion.VERSION_11
-        isCoreLibraryDesugaringEnabled = true
+        isCoreLibraryDesugaringEnabled = false
```

Comment:
@Br-DiorD Is this necessary? Curious to hear what kind of issues you were seeing when it was enabled.

Reply:
@br-jared-stockton I can't remember what the failure was but I'm trying repro it now to let you know.

Reply:
@br-jared-stockton can't repro the failure we made this change to fix maybe @BottleRocket-Colin can remember but I'm not sure. I'll revert for now since all seems well.